### ISSUE TITLE
Mutations intended for one workflow are received by others

### ISSUE THREAD

**Issue by MetRonnie** [cylc/cylc-uiserver#269]
Mutations intended for a particular workflow are received by other running workflows and they respond to them.

With the following mutation in graphiQL (or presumably by doing hold from the UI too):
```graphql
mutation {
  hold (
    workflows: ["temp4"]
    tasks: ["foo.3"]
  ) {
    result
  }
}
```

With just the hub running and no workflows running, the response is:
```json
{
  "data": {
    "hold": {
      "result": []
    }
  }
}
```
With the workflows `tutorial-family-triggers` and `temp4` running:
```json
{
  "data": {
    "hold": {
      "result": [
        "E",
        "r",
        "r",
        "o",
        "r",
        ":",
        " ",
        "N",
        "o",
        " ",
        "m",
        "a",
        "t",
        "c",
        "h",
        "i",
        "n",
        "g",
        " ",
        "W",
        "o",
        "r",
        "k",
        "f",
        "l",
        "o",
        "w",
        {
          "id": "rdutta|temp4",
          "response": [
            true,
            "Command queued"
          ]
        }
      ]
    }
  }
}
```

If you replace
https://github.com/cylc/cylc-flow/blob/c01278beed55b3275c2cd836dc20c6ecc907435a/cylc/flow/network/resolvers.py#L463

with `raise ValueError("somehting")` then you get a `GraphQLLocatedError` traceback in the terminal running the other workflow (`tutorial-family-triggers` in this case).

**Pull requests welcome!**
This is an Open Source project - please consider contributing a bug fix
yourself (please read `CONTRIBUTING.md` before starting any work though).

**Comment by MetRonnie** [comment]
Note I am looking into the separate issue of why the error message gets broken up into individual characters as part of #271

**Comment by dwsutherland** [comment]
Yeah, it's likely that the error message is in a different structure than the non-error.. The reason for this discrepancy in output formats is the attempted mapping of outputs, i.e.:

`scheduler-graphql` => `uis-graphql` => UI
`uis-graphql` => UI

So the mutation at the UIS might get a result from the UIS internals or the scheduler graphql output..
